Buildings/Fluid/Air/BaseClasses/ReheatControl.mo

```diff
@@ -0,0 +1,115 @@
+within Buildings.Fluid.Air.BaseClasses;
+model ReheatControl "Electric heater on/off controller"
+  extends Modelica.Blocks.Icons.PartialBooleanBlock;
+
+  parameter Real yValLow(min=0, max=1, unit="1")
+  "if yVal<=yValLow, hysVal.y switches to false";
```

Comment:
When adding comment, the first character should be upper case. Change `if yVal<=yValLow, hysVal.y switches to false` to `If yVal<=yValLow, hysVal.y switches to false'